Amazon.IonDotnet.Tests/TestDat/text/annot_zerosymbol.ion

```diff
@@ -0,0 +1 @@
+﻿$0::18
```

Comment:
There's really no need to have external files for these. I recommend just defining them inline in the tests. That's much easier to maintain, and makes the tests easier to read.

Reply:
Removed files